packages/twenty-server/src/engine/workspace-manager/workspace-sync-metadata/constants/standard-field-ids.ts

```diff
@@ -251,7 +251,7 @@ export const OPPORTUNITY_STANDARD_FIELD_IDS = {
   name: '20202020-8609-4f65-a2d9-44009eb422b5',
   amount: '20202020-583e-4642-8533-db761d5fa82f',
   closeDate: '20202020-527e-44d6-b1ac-c4158d307b97',
-  probability: '20202020-69d4-45f3-9703-690b09fafcf0',
+  probability_deprecated: '20202020-69d4-45f3-9703-690b09fafcf0',
```

Comment:
Should we use probabilityDeprecated instead to remain consistent?

Reply:
oops sure, thanks

Reply:
I saw Address_old was this way also (and did a quick PR fix for database:reset, make sure this PR doesn't break it also), maybe still time to fix it for Address

Reply:
oops yes still time thanks for catching this